test/e2e/storageclass.yaml

```diff
@@ -0,0 +1,6 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: ebs.csi.aws.com
+provisioner: ebs.csi.aws.com
+volumeBindingMode: WaitForFirstConsumer
```

Comment:
you might want to remove these files, or the whole test dir

Reply:
Removed whole test directory